src/state/queries/feed.ts

```diff
@@ -177,12 +177,7 @@ export function useIsFeedPublicQuery({uri}: {uri: string}) {
         })
         return Boolean(res.data.feed)
       } catch (e: any) {
-        const msg = e.toString() as string
-
-        if (msg.includes('missing jwt')) {
-          return false
-        } else if (msg.includes('This feed requires being logged-in')) {
-          // e.g. https://github.com/bluesky-social/atproto/blob/99ab1ae55c463e8d5321a1eaad07a175bdd56fea/packages/bsky/src/feed-gen/best-of-follows.ts#L13
+        if (e?.status === 401) {
           return false
         }
```

Comment:
This is an `XRPCError`, but I'm not able to import it from `@atproto/xrpc` safely due to a dependency on the node `crypto` module.